xknx/devices/climate.py

```diff
@@ -59,22 +110,27 @@ def __init__(self,
             group_address_state=group_address_temperature,
             device_name=self.name,
             after_update_cb=self.after_update)
-        self.target_temperature = RemoteValueTemp(
+        self._target_temperature = RemoteValueTemp(
             xknx,
             group_address_target_temperature,
             group_address_target_temperature_state,
             device_name=self.name,
             after_update_cb=self.after_update)
-        self.setpoint_shift = RemoteValue1Count(
+        self._base_temperature = RemoteValueTemp(
```

Comment:
> _base_temperature RemoteValueTemp() is introduced to reflect an immutable base temperature used together with setpoint_shift. It can be read from the bus if the actor supports it.

I dont even know if there are actors that provide this value as communication object. I will edit it out again to remove the bloat.
_target_temperature can then be changed to target_temperature again.